eox_core/edxapp_wrapper/backends/users_h_v1.py

```diff
@@ -197,7 +197,9 @@ def get_enabled_source_methods(cls):
     @staticmethod
     def fetch_from_created_on_site_prop(user, domain):
         """ fetch option """
-        return UserAttribute.get_user_attribute(user, 'created_on_site') == domain
+        if domain:
```

Comment:
This could be if not domain: return False, and leave the UserAttribute out of the if, perhaps in the future should do something else. @felipemontoya

Reply:
The same thing as I am doing here. Is that some prefered pythonic pattern or something?

Reply:
It's not a PEP8 standard, but it will help when the function grows and will keep the code more readable, since, you are catching the incorrect doamin value fisrt of all, and then, you can put the rest of code outside of the if. https://docs.python-guide.org/writing/style/#returning-values

Reply:
Done